develop/reference_ddl.rst

```diff
@@ -138,7 +138,9 @@ If you would prefer a table to be in its own co-location group, specify :code:`'
 
   SELECT create_distributed_table('A', 'foo', colocate_with => 'none');
 
-To co-locate a number of tables, distribute one and then put the others into its co-location group. For example:
+Using co-location group ``'none'`` can potentially take advantage of more worker nodes than existing groups. A table distributed into an existing co-location group will be placed on exactly the same nodes as other tables in the group, which doesn't include nodes added after the group was created.
```

Comment:
I wouldn't frame it like that, given we can rebalance shards to take advantage of other nodes. I'd just say, if a table isn't related to others, just use 'none'. If not we might implicitly colocate with other tables, and can have adverse impacts. E.g. when rebalancing, we might move more than just that one shard. Or for shard placement, we'll be limited to locations of the other colocated table.